src/components/TooltipTrigger.js

```diff
@@ -16,14 +16,17 @@ const baseToolTip = {
 
 const styles = {
   tooltip: {
-    background: 'black', color: 'white',
-    padding: '0.5rem',
+    background: '#FFFFFF',
```

Comment:
I am thrilled to see the tooltip color changing… I hated the black. However, something needs to be done to make the arrow pointer render better.

![image](https://user-images.githubusercontent.com/484484/180017825-69011b42-4cc2-4519-9d51-fef83fc69c47.png)

![image](https://user-images.githubusercontent.com/484484/180017897-56af694c-d7b1-45cf-9d3c-44caa8927742.png)

Reply:
@n8landolt I will take on fixing this issue in https://broadworkbench.atlassian.net/browse/TUIM-23. Please back out the change for now.